NOnion/Cells/ICell.fs

```diff
@@ -1,5 +1,9 @@
 ﻿namespace NOnion.Cells
 
+open System.IO
```

Comment:
please follow our CONTRIBUTING guidelines, this namespace is used only once in the file so it has to be inline

Reply:
Will fix and submit a PR to suggest removing this guideline as a requirement.